app/views/posts/index.html.haml

```diff
@@ -0,0 +1,28 @@
+.container
+  - if @posts.any?
+    - @posts.reverse.each do |post|
+      .well.well-lg
+        %p.username= post.email
+        = link_to (image_tag post.image.url(:large)), post_path(post), id: post.id
+        %p.caption= post.caption
+        = link_to post_likes_path(post), class: 'likes-link', id: 'Like' do
+          %span.glyphicon.glyphicon-heart
+        %span.likes_count= post.likes.count
+        = 'like'.pluralize post.likes.count
+        - if current_user
+          - if (post.user_id == current_user.id)
+            %p= link_to "Delete Post", post_path(post), method: :delete
+        %legend
+        - if post.comments.any?
+          - post.comments.each do |comment|
+            %li
+              = "#{User.find(comment.user_id).email} "
+            %br
+            %p
+              = comment.comments
+        - else
+          %p No Comments
+    %p= link_to "Add a Post", new_post_path
+  - else
+    No Pictures Yet!
+    %a(href='/posts/new') Add a Post
```

Comment:
Prefer the hash attributes syntax (%tag{ lang: 'en' }) over HTML attributes syntax (%tag(lang=en))